src/app/groups/models/group-arrpovals.ts

```diff
@@ -0,0 +1,24 @@
+import * as D from 'io-ts/Decoder';
+import { dateDecoder } from 'src/app/utils/decoders';
+
+export const groupApprovalsDecoder = D.struct({
+  requireLockMembershipApprovalUntil: D.nullable(dateDecoder),
+  requirePersonalInfoAccessApproval: D.literal('none', 'view', 'edit'),
+  requireWatchApproval: D.boolean,
+});
```

Comment:
next time, when you change a decoder, you should take the opportunity to convert it to zod

Reply:
oh, yes, I've missed it